hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/HapiSuite.java

```diff
@@ -227,19 +235,20 @@ private FinalOutcome runSuite(final Consumer<List<HapiSpec>> runner) {
         }
 
         List<HapiSpec> specs = getSpecsInSuite();
+        final var name = name();
         for (final var spec : specs) {
             if (!overrides.isEmpty()) {
                 spec.addOverrideProperties(overrides);
             }
+            spec.setSuitePrefix(name);
```

Comment:
BTW, what is the suite "prefix" for?

Reply:
I have no idea.  I just moved it from a second lambda executed in a stream into the loop so we don't loop twice for no reason.